src/app/modules/library/learning-resources/learning-resources.component.scss

```diff
@@ -0,0 +1,50 @@
+.c-filters {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: white;
+}
+
+.c-card {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 5%;
+}
+
+.example-header-image {
+  background-image: url("https://material.angular.io/assets/img/examples/shiba1.jpg");
+  background-size: cover;
+}
+
+.c-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 300px;
+  min-width: 300px;
+  flex-direction: column;
+  margin: 10px;
+}
+.icon-background {
+  background-color: #e5f3fe;
+  height: 70px;
+  width: 70px;
+  border-radius: 35px;
+  margin: 20% 35% 10% 35%;
+  // padding: 20%;
+}
+
+.whole-card {
+  margin-top: 3%;
+}
+
+.c-image {
+  width: 30px;
+  margin-top: 25px;
+}
+.c-description {
+  text-align: center;
+}
+
+.c-view {
+  padding: 5% 0% 0% 50%;
```

Comment:
Why is 50% here?

Reply:
For view all button to place in middle of the cards

Reply:
changed to bootstrap